pywinauto/hooks.py

```diff
@@ -4,41 +4,45 @@
 
 
 def create_pointer(handler):
+	"""Create and return C-pointer"""
```

Comment:
Seems like different indentation. Maybe forgot to set up `Replace tabs with spaces` in your editor?